### PR TITLE
deps bump for hbs strategy

### PR DESCRIPTION
**Pull request by betocantu93**
These are the updated deps needed for #186, splitted  as requested here  #166

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -62,14 +62,12 @@ jobs:
     strategy:
       matrix:
         scenario:
-          - ember-lts-2.8
-          - ember-lts-2.12
-          - ember-lts-2.16
-          - ember-lts-3.4
-          - ember-lts-3.8
+          - ember-lts-3.16
+          - ember-lts-3.20
```
Comment (jherdman) [inline .github/workflows/test.yml:66]: @ivanvotti thoughts? I think this is okay personally given that many of these aren't even receiving support from Ember Core these days.

File: eslint-rules.js [hunk 2]
```diff
@@ -15,22 +15,28 @@ module.exports = {
     'import/extensions': 'off',
     'import/no-dynamic-require': 'off',
     'prefer-const': 'off',
+    'lines-between-class-members': 'off',
+    'template-curly-spacing': 'off',
+
+    indent: ['error', 2, {
+      ignoredNodes: ['TemplateLiteral']
+    }],
```
Comment (jherdman) [inline eslint-rules.js:23]: Can you comment on why these were added?

File: package.json [hunk 3]
```diff
@@ -101,7 +108,7 @@
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged",
-      "pre-push": "yarn test"
+      "pre-push": "yarn lint:js && yarn test:node"
```
Comment (jherdman) [inline package.json:111]:
```suggestion
      "pre-push": "yarn lint && yarn test:node && yarn test:ember"
```

**Review by jherdman — CHANGES_REQUESTED** [review]

File: eslint-rules.js (hunk 2 above)
Comment (betocantu93) [reply eslint-rules.js]: Woops, I think I blindly rushed through the lint error, my bad. We don't actually need it.

File: .github/workflows/test.yml (hunk 1 above)
Comment (ivanvotti) [reply .github/workflows/test.yml:66]: That's fine. Though, it would be good to add a note about this to the Readme.

Comment (jherdman) [reply .github/workflows/test.yml:66]: https://github.com/ivanvotti/ember-svg-jar/blob/master/README.md#-compatibility << i.e. this should be updated.

I think it ought to say "Any active Ember v3 LTS, or release."

**Review by jherdman — CHANGES_REQUESTED** [review]
Quick suggestion for the README.

**Comment by betocantu93** [comment]
@jherdman updated readme

**Review by jherdman — APPROVED** [review]

**Comment by jherdman** [comment]
Many thanks for this effort! I know it wasn't easy.

**Pull request merged**